problem_details.py

```diff
@@ -0,0 +1,31 @@
+from core.util.problem_detail import ProblemDetail as pd
```

Comment:
Stolen wholesale from the library registry.

Reply:
I've changed these in a recent registry branch. You should be able to eliminate some of them by going direct to the document with the public key in it.